### DISCUSSION TITLE
🔴 Free YouTube Video

### DISCUSSION THREAD

**Discussion by rebeloper** [SwifterSwift/SwifterSwift#934] [General]
Hi,
I have been using SwifterSwift for a while now, including in live apps. I like it so much that I have made a YouTube video explaining how it works. Just the basics, but it helps to set up SwifterSwift and promote it 👍 
Here it is: https://www.youtube.com/watch?v=Gi47Oi8gKJE&list=PL_csAAO9PQ8Zia0zTZZjd0KZBEkcKb4yp
If you like it feel free to add it to the README file. I know that people click on Images more often than on Links so you could add in this image also:
![SwiftExtensions](https://user-images.githubusercontent.com/2488011/103780693-fe9f1e00-503d-11eb-8d6e-764ae19c5a2e.png)

Thanks for the awesome repo!

**Comment by QF92** [comment]
good
